<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window_RenewBook</class>
 <widget class="QDialog" name="Window_RenewBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget_RentedBooks">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>401</width>
     <height>341</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>BookName</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Subject</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Author</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>RentedUntil</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>251</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Book ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_BookId"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Member ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_MemberId"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_search">
      <property name="text">
       <string>search</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>350</y>
     <width>244</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Renewd Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_renewdDate"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Renew">
      <property name="text">
       <string>Renew</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
